### ISSUE TITLE
Idea: Swing through trees with your tail

### ISSUE THREAD

**Issue by skylerberg** [SageBerg/St.GeorgeGame#14]
If you have a tail, you should be able to try to swing through the trees in the woods using it. Perhaps this could be a way to escape bad stuff in the woods. Otherwise, I am not sure if it will be useful, but it would make having a tail more interesting.

**Comment by SageBerg** [comment]
Yeah, having a tail doesn't do much, so adding more content for it is on my to do list.

Here are some ideas I'm coming up with just now:
-Maybe the wizard could see your tail and say you look like a man with an appreciation for magic and then cast some other spell on you or give you a potion.
-You could slam your tail in the door when leaving buildings.
-Your tail (or a bouquet of flowers) could be required to marry Olga.
